Http code is changed from 200 to 202 for scan request submitting, scan all is still using 200

If you are reporting a problem, please make sure the following information are provided:

**Expected behavior and actual behavior:**

No errors logged but now many error logs appeared in the log file

**Steps to reproduce the problem:**

Just trigger a scan action

**Versions:**
Please specify the versions of following systems.

- harbor version: [1.10.0]
- docker engine version: [y.y.y]
- docker-compose version: [z.z.z]

**Additional context:**

> Oct 29 00:00:00 192.168.176.1 jobservice[1185]: 2019-10-29T00:00:00Z [ERROR] [/jobservice/runner/redis.go:191]: Unexpected response code: 202, data: []
Oct 29 00:00:01 192.168.176.1 jobservice[1185]: 2019-10-29T00:00:01Z [ERROR] [/jobservice/runner/redis.go:191]: Unexpected response code: 202, data: []
Oct 29 00:00:01 192.168.176.1 jobservice[1185]: 2019-10-29T00:00:01Z [ERROR] [/jobservice/runner/redis.go:191]: Unexpected response code: 202, data: []
Oct 29 01:50:41 192.168.176.1 jobservice[1185]: 2019-10-29T01:50:41Z [ERROR] [/jobservice/runner/redis.go:191]: Unexpected response code: 202, data: []
Oct 29 01:50:41 192.168.176.1 jobservice[1185]: 2019-10-29T01:50:41Z [ERROR] [/jobservice/runner/redis.go:191]: Unexpected response code: 202, data: []
Oct 29 01:50:41 192.168.176.1 jobservice[1185]: 2019-10-29T01:50:41Z [ERROR] [/jobservice/runner/redis.go:191]: Unexpected response code: 202, data: []

--- a/src/jobservice/job/impl/scan/all.go
+++ b/src/jobservice/job/impl/scan/all.go
@@ -95,7 +95,7 @@ func (sa *All) Run(ctx job.Context, params job.Parameters) error {
 				data, err := ioutil.ReadAll(resp.Body)
 				if err != nil {
 					logger.Errorf("Failed to read response, error: %v", err)
-				} else if resp.StatusCode != http.StatusOK {
+				} else if resp.StatusCode != http.StatusAccepted {
 					logger.Errorf("Unexpected response code: %d, data: %v", resp.StatusCode, data)
 				}
 				resp.Body.Close()

--- a/src/pkg/scan/event/init.go
+++ b/src/pkg/scan/event/init.go
@@ -26,5 +26,7 @@ func Init() {
 	log.Debugf("Subscribe topic %s for cascade deletion of scan reports", model.DeleteImageTopic)
 
 	err := notifier.Subscribe(model.DeleteImageTopic, NewOnDelImageHandler())
-	log.Error(errors.Wrap(err, "register on delete image handler: init: scan"))
+	if err != nil {
+		log.Error(errors.Wrap(err, "register on delete image handler: init: scan"))
+	}
 }

